agents/check_mk_agent.macosx

```diff
@@ -281,7 +306,7 @@ then
             DNAME="$(smartctl -d nvme -i -f brief $REPLY | grep -v 'Family' | grep -E 'Model|^Serial Number' | sed 's/^.*\(  .*\).*$/\1/' | tr '\n' '_' | sed -e 's/  //g' -e 's/ /_/g' -e 's/_$//' -e 's/^APPLE_//')"
             if [ -n "${DNAME}" ]; then
                 # NVME
-                echo "$DNAME NVME $(sed 's/\_[^.]\{0,15\}$//g' <<<${DNAME})"
+                echo "$REPLY NVME $(sed 's/\_[^.]\{0,15\}$//g' <<<${DNAME})"
```

Comment:
@ThomasKaiser : Found a new unrelated bug for NVME drives in OSX. It was double printing DNAME (ie. model name) instead of REPLY (ie. drive name)